modules/performance.nix

```diff
@@ -4,6 +4,24 @@ let
 in
 {
   options.performance = {
+    byteCompileLua = {
+      enable = lib.mkEnableOption "byte compiling of lua files";
+      initLua = lib.mkEnableOption "initLua" // {
+        description = "Whether to byte compile init.lua.";
+        default = true;
+      };
+      configs = lib.mkEnableOption "configs" // {
+        description = "Whether to byte compile lua configuration files.";
+        default = true;
+      };
```

Comment:
I think it'd be clearer to not use mkEnableOption here, as I feel that it should be reserved for option with default = false